paasta_tools/kubernetes_tools.py

```diff
@@ -158,6 +158,7 @@
 log = logging.getLogger(__name__)
 
 KUBE_CONFIG_PATH = "/etc/kubernetes/admin.conf"
+DEFAULT_KUBECTL_CONFIG_PATH = "/etc/kubernetes/paasta.conf"
```

Comment:
nit: maybe rename this to something that makes it clearer why there's two kubeconfig files, like `KUBE_CONFIG_HUMAN_PATH` or `KUBE_CONFIG_USER_PATH`